ipam/actor.go

```diff
@@ -0,0 +1,165 @@
+package ipam
```

Comment:
I wouldn't mind seeing a split along functionality lines. See my comments on claims.go.

As an aside, if we do want to split that "client" and "server" part of the actor, then the *client* part, including the New* constructor, should be what lives in the <actor-name>.go since the functions there define the API; the fact that the API is implemented in terms of an actor is an implementation detail.